src/rcms/fm/app/level1/HCALParameters.java

```diff
@@ -85,10 +85,10 @@ public synchronized void initializeParameters() throws ParameterException {
 
 		this.put( new FunctionManagerParameter<StringT>  ("HCAL_RUN_TYPE"                    ,  new StringT("local")      ) );  // Run type -- local or global
 		this.put( new FunctionManagerParameter<StringT>  ("HCAL_COMMENT"                     ,  new StringT("")           ) );  // User-input comment
-		this.put( new FunctionManagerParameter<MapT<MapT<StringT>>>  ("AVAILABLE_RUN_CONFIGS"  ,  new MapT<MapT<StringT>>()));  // Local run types available
-		this.put( new FunctionManagerParameter<VectorT<StringT>> ("AVAILABLE_LOCALRUNKEYS"   ,  new VectorT<StringT>()    ) );  // List of local run keys
-		this.put( new FunctionManagerParameter<StringT>  ("RUN_CONFIG_SELECTED"              ,  new StringT("not set")    ) );  // User selected local run type
-		this.put( new FunctionManagerParameter<StringT>  ("CFGSNIPPET_KEY_SELECTED"          ,  new StringT("not set")    ) );  // Key name for the local run type selected by the user (aka Mastersnippet file)
+		this.put( new FunctionManagerParameter<MapT<MapT<StringT>>>  ("LOCAL_RUNKEY_MAP"     ,  new MapT<MapT<StringT>>()));    // Local run types available
+		this.put( new FunctionManagerParameter<VectorT<StringT>> ("AVAILABLE_LOCAL_RUNKEYS"  ,  new VectorT<StringT>()    ) );  // List of local run keys
+		this.put( new FunctionManagerParameter<StringT>  ("MASTERSNIPPET_SELECTED"           ,  new StringT("not set")    ) );  // User selected mastersnippet
+		this.put( new FunctionManagerParameter<StringT>  ("LOCAL_RUNKEY_SELECTED"            ,  new StringT("not set")    ) );  // Key name for the local run type selected by the user (aka Mastersnippet file)
```

Comment:
suggest
`LOCAL_RUNKEY_MAP` to have comment of `//Local run types available: <LocalRunKey name : <LocalRunKey attribute: Value>`
`AVAILABLE_LOCAL_RUNKEYS` to have comment of `//List of local run keys names, keys of LOCAL_RUNKEY_MAP`
`MASTERSNIPPET_SELECTED` to have comment of `//User selected mastersnippet file name`
`LOCAL_RUNKEY_SELECTED` to have comment of `//Key name for the local run key selected by the user`